framework/src/modules/interoperability/base_interoperability_endpoint.ts

```diff
@@ -31,19 +31,22 @@ import { TerminatedOutboxAccountJSON } from './stores/terminated_outbox';
 import { BaseInteroperabilityStore } from './base_interoperability_store';
 import { chainAccountToJSON } from './utils';
 import { ImmutableStoreGetter, StoreGetter } from '../base_store';
+import { MainchainInteroperabilityStore } from './mainchain/store';
+import { SidechainInteroperabilityStore } from './sidechain/store';
```

Comment:
hmm its better if we don't reference to Mainchain or sidechain as they are derived from these base classes, else the base class will depend on these classes that derived.
Maybe we can switch to instance member property instead of member?
Something like,
```ts
protected abstract getInteroperabilityStore: (context: StoreGetter | ImmutableStoreGetter) => T;
```